infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/EndpointMetadata.java

```diff
@@ -272,7 +295,20 @@ public EndpointMetadata build() {
         withInternalErrorResponse();
       }
       return new EndpointMetadata(
-          method, path, operationId, summary, security, description, responses, requestBodyType);
+          method,
+          path,
+          operationId,
+          summary,
+          security,
+          description,
+          responses,
+          requestBodyType,
+          tags);
+    }
+
+    public EndpointMetaDataBuilder tags(final List<String> tags) {
+      this.tags = tags;
```

Comment:
Make things simpler for callers:
```suggestion
    public EndpointMetaDataBuilder tags(final String... tags) {
      this.tags = List.of(tags);
```

Reply:
nice. i like it.